test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -352,6 +355,18 @@ void ProtoConverter::appendTypedParamsPublic(
 		.render();
 }
 
+void ProtoConverter::appendToIsabelleTypeString(
+	std::string const& _typeString,
+	Delimiter _delimiter
+)
+{
+	m_isabelleTypeString
+		<< Whiskers(R"(<delimiter><typeString>)")
+		("delimiter", delimiterToString(_delimiter))
+		("typeString", _typeString)
```

Comment:
The overhead of runtime regex replacement for this rather than just using ``delimiterToString(_delimiter) + _typeString`` is probably not justified ;-). And for the other whisker uses it's at least borderline - be aware that the whiskers template will stay there at runtime and it'll actually do runtime regex replaces, so in comparison to simple concatenation it's extremely expensive, so maybe worth it to reduce the use of fuzzing resources a bit there.

Reply:
Thanks, updated source

Reply:
@ekpyron If you are okay with the changes, could you please re approve it. I will then rebase a follow up PR (for values) on top of this one.